### ISSUE TITLE
[Adapter Cards-Schema Explorer]: In scan mode user is not able to access the list items present under the adaptive cards using arrow key as focus is not synchronising with the narrator announcement.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7854]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Screen Reader: Narrator
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/](https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Pre-Requisites:
Run the Narrator using "Control+ Windows +Enter" short key in the scan mode.

Repro Steps:
1.Open above URL with V- credential.
2.Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
3.Navigate to 'Input.Text" link present under Input from the left navigation using the tab key and activate it using Enter key.
4. Navigate to the adaptive cards present under it using the Tab key.
5. Navigate to the "Required Input.ChoiceSet Label" combo box under Adaptive Cards and activate using the Enter key.
6. Try to navigate through the list items present under it using narrator in the scan mode.
7. Observe the issue here.

Actual Result:
User is not able to Access the list items present under the "Input.ChoiceSet Label(Compact)" combo box in the scan mode as focus is not in sync with the narrator announcement.

Expected Result:
User should be able to access and activate the list items present under the "Input.ChoiceSet Label" combo box using the arrow keys and focus should be in syn with the narrator announcement.

Observation:
1.Same issue exists throughout the application wherever the similar options present in the adaptive cards.
2.Similar issue is present under the "Input.Date" link present under the input from the left navigation pane.
PFA: 2.Focus is not in syn with the narrator announcemnt in the calender..mp4.

User Experience:
Screen reader users who are with low vision will get impacted if keyboard focus is not in sync with Narrator after which user will not be able to track or navigate the content properly.

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.02

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/104482612/190184701-fc336b5a-0299-4876-963f-da3bd75803a5.mp4
2.https://user-images.githubusercontent.com/104482612/190193151-b5b7caf3-16ab-4f3b-8f79-5c84c94fd84f.mp4

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.
